### PR TITLE
Go wrapper updates

### PR DESCRIPTION
**Pull request by atifali**
Following two additions to the Go wrapper:

- Bit Accessors 
- Constants for the Debug Levels

### CODE REVIEW

File: src/wrappers/go/src/plctag/plctag.go [hunk 1]
```diff
@@ -30,46 +30,53 @@ import "unsafe"
...
-	ERR_REMOTE_ERR = C.PLCTAG_ERR_REMOTE_ERR
-	ERR_THREAD_CREATE = C.PLCTAG_ERR_THREAD_CREATE
-	ERR_THREAD_JOIN = C.PLCTAG_ERR_THREAD_JOIN
-	ERR_TIMEOUT = C.PLCTAG_ERR_TIMEOUT
-	ERR_TOO_LARGE = C.PLCTAG_ERR_TOO_LARGE
-	ERR_TOO_SMALL = C.PLCTAG_ERR_TOO_SMALL
-	ERR_UNSUPPORTED = C.PLCTAG_ERR_UNSUPPORTED
-	ERR_WINSOCK = C.PLCTAG_ERR_WINSOCK
-	ERR_WRITE = C.PLCTAG_ERR_WRITE
-	ERR_PARTIAL = C.PLCTAG_ERR_PARTIAL
```
Comment (atifali) [inline src/wrappers/go/src/plctag/plctag.go:72]: woops - looks like my gofmt on save doesn't like spacing here!

**Comment by kyle-github** [comment]
Thanks for this!  I changed the base to the `prerelease` branch so that I can run it through the CI tests first.   I am seeing some failures on the ARM builds.   Let's see if it reproduces on a rerun.

Are you OK with licensing these changes under the LGPL and MPL?  You keep the copyright, but I want to make sure that you are OK with that!

**Comment by kyle-github** [comment]
Huh.  Looks like the GitHub Actions executors might not be stable.  I put in some extra code in the CI scripts to make sure that all packages are updated before installing anything.   I then had another failure in the CI which looks like a mis-compilation.  

I finally got a clean CI run of the `prerelease` branch, so I will kick off these tests for this PR again.

**Comment by atifali** [comment]
Yeah I am ok with the licensing you mentioned. Thanks for the help with the CI builds! I was stumped there for a while too!

**Comment by kyle-github** [comment]
I got a clean run.   As soon as you get back to me with an answer on the license, I can merge this!

**Comment by kyle-github** [comment]
Oops, crossed posts!

I will merge and send out a release today.

Thanks again for this!

If you are interested in doing this more independently, I can set up a new project in the libplctag organization for Go.  Right now I have Java, Julia and C# broken out and another related repo for Android.   Tim and Jody are doing a great job keeping the C# repo moving along and adding all kinds of nice C# features to it.   Keith and his group are supporting Julia really well too.   

I want the wrapper to be more than just a thin layer over the C library.   Ideally they should provide a "native" feel to how to use the C library.

Let me know if you are interested.

**Comment by atifali** [comment]
Cool! At this point the changes I am making are mostly to access the functionality that is missing for my other projects. 

Once I am at a point where this needs to be a stand-alone Go pkg I think a separate repo would make more sense. I will let you know!

**Pull request merged**